docker-compose.yml

```diff
@@ -17,7 +17,7 @@ services:
       - ./data/redis/database_data:/data
 # BBB LTI BROKER ↓:
   broker:
-    image: bigbluebutton/bbb-lti-broker
+    image: bigbluebutton/bbb-lti-broker:beta
```

Comment:
This is the code we use for releases, it should not be linked to beta. In any case this must be done by the scripts on deployment. What would happen when we release as stable?